.fixtures.yml

```diff
@@ -5,10 +5,7 @@ fixtures:
     apt:
       repo:           'git://github.com/puppetlabs/puppetlabs-apt'
       branch:         '1.8.x'
-    concat:
-      repo:           'git://github.com/puppetlabs/puppetlabs-concat'
-      branch:         '1.2.x'
-    file_concat:      'git://github.com/electrical/puppet-lib-file_concat'
+    concat:           'git://github.com/puppetlabs/puppetlabs-concat'
```

Comment:
It claims to, but it's actually compatible.  I just opened a PR to relax the dep and fix a couple of minor deprecations.